proposals/0001-simd-process.md

```diff
@@ -0,0 +1,170 @@
+---
+simd: '0001'
+title: Solana Proposal Process
+authors:
+  - Jacob Creech (Solana Foundation)
+category: Meta
+type: Meta
+status: Draft
+created: 2022-10-18
+---
+
+## What is a Proposal?
+
+A proposal is a design document describing a new feature for Solana or its
+processes or environment. The proposal should document the rationale for the
+feature and enough documentation to understand the implementation.
+
+## Rationale
+
+A proposal is intended to be reviewed by core engineering and community members
+keeping in mind security concerns, tradeoffs, and backwards compatibility.
+Having a proposal process helps identify design issues early, alert the
+community on a change, helps scale newer contributors on the architecture, and
+acts as a historical record of the design decisions that have gone into Solana.
+For implementors, the proposal is a blueprint for the feature and helps track
+the development of a feature.
+
+## When you need to follow this process
+
+You need to follow this process if you intend to make "substantial" changes to
+the Validator, RPC, consensus, or a change to the proposal process itself. What
+constitutes a "substantial" change is evolving based on community norms and
+varies depending on what part of the ecosystem you are proposing to change, but
+may include the following:
+
+- A change in format of a RPC API method
+- Networking interface changes between validators
+- Compute requirement changes on the runtime
+
+Some changes do not require a proposal:
+
+- Rephrasing, reorganizing, refactoring, or otherwise "changing shape does not
+change meaning".
+- Additions that strictly improve objective, numerical quality criteria
+(warning removal, speedup, better platform coverage, more parallelism, trap
+more errors, etc.)
+
+## Proposal Types
+
+There are two types of proposals:
+
+- A **Standard** Proposal describes any change that affects most or all Solana
+implementations, such as a change to the network protocol, consensus, proposed
+application standards/conventions, block or transaction validity, or any change
+or addition that affects the interoperability of applications using Solana.
+Standard proposals can be broken down into the following categories:
+
+  - **Core**: Anything that affects consensus or substantial changes to the
+validator.
+  - **Networking**: Changes or substantial improvements to network protocol
+specifications.
+  - **Interfaces**: Improvements around the client JSON RPC API specifications
```

Comment:
Could we rewrite this as "changes to the client JSON RPC API specifications and standards"? Or maybe even "breaking changes"? Seems a bit onerous as written.

Reply:
Done